nx.json

```diff
@@ -60,9 +61,37 @@
       }
     ]
   },
+  "workspaceLayout": {
+    "appsDir": "",
+    "libsDir": ""
+  },
+  "tasksRunnerOptions": {
+    "default": {
+      "runner": "@nrwl/nx-cloud",
+      "options": {
+        "cacheableOperations": [
+          "build",
+          "test",
+          "lint",
+          "package",
+          "prepare",
+          "e2e",
+          "version",
+          "config",
+          "tsc",
+          "typedoc"
+        ],
+        "useDaemonProcess": true,
+        "accessToken": "NzU3MWE3MjgtOWRiZi00ZDczLWI4YWYtMDI5NThmNTRkNDc1fHJlYWQtd3JpdGU="
```

Comment:
What is this access token used for?

Reply:
That is for nx cloud, its set by nx, been there the whole time. seems fine to be in clear text.